src/test/scala-2.11/markets/MarketsTestKit.scala

```diff
@@ -0,0 +1,54 @@
+package markets
+
+import akka.actor.ActorSystem
+import akka.testkit.{TestActorRef, TestKit}
+
+import java.util.UUID
+
+import markets.orders.Order
+import markets.orders.limit.{LimitAskOrder, LimitBidOrder, LimitOrderLike}
+import markets.orders.market.{MarketAskOrder, MarketBidOrder, MarketOrderLike}
+import markets.tradables.Tradable
+
+import scala.util.Random
+
+
+trait MarketsTestKit {
+
+  /** Generates a timestamp. */
+  def timestamp(): Long = {
+    System.currentTimeMillis()
+  }
+
+  /** Generates a UUID. */
+  def uuid(): UUID = {
+    UUID.randomUUID()
+  }
+
+  /** Returns a randomly generated limit price
+    *
+    * @param prng
+    * @param lower
+    * @param upper
+    * @return
+    */
+  def randomLimitPrice(prng: Random, lower: Long = 1, upper: Long = Long.MaxValue): Long = {
+    nextLong(prng, lower, upper)
+  }
+
+  /** Returns a randomly generated quantity.
+    *
+    * @param prng
+    * @param lower
+    * @param upper
+    * @return
+    */
+  def randomQuantity(prng: Random, lower: Long = 1, upper: Long = Long.MaxValue): Long = {
+    nextLong(prng, lower, upper)
+  }
+
+  /* Returns a randomly generated Long integer between some lower and upper bound. */
+  private [this] def nextLong(prng: Random, lower: Long, upper: Long) = {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [There should be no space before a left bracket '['](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2385917860/issues/source?bid=2921361&fileBranchId=3228019#l51)